app/v2/models/response/retrieveForeignPropertyAnnualSubmission/foreignProperty/ForeignPropertyAllowances.scala

```diff
@@ -21,10 +21,11 @@ import play.api.libs.json.{Json, OFormat}
 case class ForeignPropertyAllowances(annualInvestmentAllowance: Option[BigDecimal],
                                      costOfReplacingDomesticItems: Option[BigDecimal],
                                      zeroEmissionsGoodsVehicleAllowance: Option[BigDecimal],
```

Comment:
`zeroEmissionGoodsVehicleAllowance` not `zeroEmissionsGoodsVehicleAllowance` but is `zeroEmissionsGoodsVehicleAllowance` downstream so will need custom reads